<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(144, 196, 231)
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Delete_all">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>360</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;
</string>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>30</y>
      <width>481</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_sum">
       <property name="styleSheet">
        <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;</string>
       </property>
       <property name="text">
        <string>Сумма штрафа:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_name">
       <property name="styleSheet">
        <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;</string>
       </property>
       <property name="text">
        <string>Имя Фамилия:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="Sum">
       <property name="styleSheet">
        <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:300px;
padding:3px 7px;</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="singleStep">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Name">
       <property name="styleSheet">
        <string notr="true">color:white;
background-color:red;
border: 3px solid white;
</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="In_system">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;
</string>
    </property>
    <property name="text">
     <string>Добавить в систему</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sum_smena">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>209</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;
</string>
    </property>
    <property name="text">
     <string>Сумма штрафов за смену:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Summa_smena">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>399</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>137</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;</string>
    </property>
    <property name="text">
     <string>Имя Фамилия:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="find">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;
</string>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="find_sum">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>170</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;</string>
    </property>
    <property name="text">
     <string>Посчитать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Name_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="Sum_person">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>399</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
